packages/transports/ws/package.json

```diff
@@ -40,7 +40,7 @@
     "@graphql-mesh/string-interpolation": "^0.5.6",
     "@graphql-mesh/transport-common": "^0.7.6",
     "@graphql-mesh/utils": "^0.102.5",
-    "@graphql-tools/executor-graphql-ws": "^1.2.0",
+    "@graphql-tools/executor-graphql-ws": "1.3.0-alpha-20240925200710-d61ee3efd7a824d2ae97ac731770da8ae3a913d0",
```

Comment:
Let's bump this to the stable `1.3.0` once https://github.com/ardatan/graphql-tools/pull/6544 CI completes.